stackslib/src/config/mod.rs

```diff
@@ -1238,63 +1238,63 @@ pub struct BurnchainConfig {
     /// The underlying blockchain used for Proof-of-Transfer.
```

Comment:
this seems to have the same changes as #6168 is that correct? i would recommend that either that PR is closed, or we keep the changes in teh PR's separate.